src/api.rs

```diff
@@ -2599,3 +2698,16 @@ pub struct GitCommit {
     pub message: Option<String>,
     pub id: String,
 }
+
+#[derive(Clone, Debug, Deserialize)]
+pub struct ProcessedEvent<'a> {
```

Comment:
not that familiar with lifetimes in structs yet, why do we need a lifetime here and the `Cow`s below?